frontend/server/src/Controllers/User.php

```diff
@@ -9,7 +9,8 @@
  * @psalm-type AuthorsRank=array{ranking: list<array{author_ranking: int|null, author_score: float, classname: string, country_id: null|string, name: null|string, username: string}>, total: int}
  * @psalm-type AuthorRankTablePayload=array{length: int, page: int, ranking: AuthorsRank, pagerItems: list<PageItem>}
  * @psalm-type Badge=array{assignation_time: \OmegaUp\Timestamp|null, badge_alias: string, first_assignation: \OmegaUp\Timestamp|null, owners_count: int, total_users: int}
- * @psalm-type CommonPayload=array{omegaUpLockDown: bool, bootstrap4: bool, inContest: bool, isLoggedIn: bool, isReviewer: bool, gravatarURL51: string, currentUsername: string, userClassname: string, userCountry: string, profileProgress: float, isMainUserIdentity: bool, isAdmin: bool, lockDownImage: string, navbarSection: string}
+ * @psalm-type UsernameIdentity=array{username: string, default: bool}
```

Comment:
argh, se me fue la vez pasada. esto se puede mejor llamar

```suggestion
 * @psalm-type AssociatedIdentity=array{username: string, default: bool}
```

? "UsernameIdentity" suena a que es un tipo de identidad que es un nombre de usuario, lo cual suena raro.

Reply:
Listo